Jenkinsfile

```diff
@@ -7,13 +7,19 @@ pipeline {
     }
     stages {
         stage('Build') {
+            agent {
+                docker {
+                    image "python:3"
```

Comment:
This allows us to use python 3's venv.

Reply:
@kbridbur

I want to make sure that the work being done in the build step is not being thrown away when we move on to the next step. which does not have the same docker agent specification.